includes/Renderer/Story/Image.php

```diff
@@ -52,6 +52,7 @@ class Image {
 	 */
 	public function __construct( Story $story ) {
 		$this->story = $story;
+		add_filter( 'web_stories_render_image', 'wp_filter_content_tags' );
```

Comment:
There maybe a better place for this....

Reply:
Yeah we should not be adding filters in constructors.